src/engine/graphic/GRStaff.cpp

```diff
@@ -93,6 +93,7 @@ using namespace std;
 #include "kf_ivect.h"
 #include "TCollisions.h"
 
+int gd_noteName2pc(const char *name);
```

Comment:
this declaration is useless (and prone to conflicts), the function is declared in guido.h

Reply:
I see declarations of this function in multiple places in .cpp files. I guess the reason is that I - and other developers before me - just add all Guido files and all LibMusicXML files to one Xcode project. In LibMusicXML, there is another guido.h file, which sometimes wrongly get included instead of the GuidoLib guido.h. Would renaming guido.h and guido.cpp be an option?

Reply:
ok, I'll rename these files asap